<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioEffectVolumeWidget</class>
 <widget class="QWidget" name="AudioEffectVolumeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>51</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="effectNameLabel">
     <property name="text">
      <string notr="true">Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="volumeWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSlider" name="volumeSlider">
        <property name="styleSheet">
         <string notr="true">QSlider:vertical {
	width:30px;
}
QSlider::groove:vertical {
	width:4px;
	border: 3px solid rgb(30, 30, 30);
	border-color: rgb(25, 25, 25) rgb(60, 60, 60) rgb(60, 60, 60) rgb(30, 30, 30);
	background-color: black; 
	border-radius: 3px;
}


QSlider::handle:vertical {
	height: 45px;
	margin: 0 -23px;
	image: url(:/icons/images/mixer/slider-v.svg);
}</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>74</number>
        </property>
        <property name="sliderPosition">
         <number>74</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="VUMeter" name="vumeter" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="muteButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VUMeter</class>
   <extends>QWidget</extends>
   <header>vumeter.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
